lib/mix/tasks/compile.make.ex

```diff
@@ -83,6 +83,11 @@ defmodule Mix.Tasks.Compile.ElixirMake do
   and `mix test` commands.
   """
 
+  @exec_notfound_msg """
```

Comment:
Maybe, actually this one is annoying to have separated because it requires a dynamic component (the executable) so the message has to be used like `executable <> @exec_notfound_msg`. But since it would be the only error message inlined we can keep it like this for now and change later if we want to :)